### PR TITLE
Convert UIWebView to WKWebView. Allows compilation in apps targetting UIKit for Mac.

### PR DESCRIPTION
**Pull request by colinhumber**

### CODE REVIEW

**Comment by NSExceptional** [comment]
I apologize, I must have overlooked this PR somehow! An equivalent PR (#296) has been merged now, however, so I'm going to close this.

File: FLEX.podspec [hunk 1]
```diff
@@ -32,7 +32,7 @@ Pod::Spec.new do |spec|
   spec.platform         = :ios, "8.0"
   spec.source           = { :git => "https://github.com/Flipboard/FLEX.git", :tag => "#{spec.version}" }
   spec.source_files     = "Classes/**/*.{h,m,mm}"
-  spec.frameworks       = [ "Foundation", "UIKit", "CoreGraphics", "ImageIO", "QuartzCore" ]
+  spec.frameworks       = [ "Foundation", "UIKit", "CoreGraphics", "ImageIO", "QuartzCore", "WebKit" ]
```
Comment (revolter) [inline FLEX.podspec]: @NSExceptional, I did not add this in my PR though.

**Comment by NSExceptional** [comment]
I added it.

File: FLEX.podspec (hunk 1 above)
Comment (NSExceptional) [reply FLEX.podspec]: I added it.

**Pull request closed without merging**